packages/backend/src/server/api/endpoints/miauth/gen-token.ts

```diff
@@ -71,6 +73,9 @@ export default class extends Endpoint<typeof meta, typeof paramDef> { // eslint-
 				permission: ps.permission,
 			});
 
+			// アクセストークンが生成されたことを通知
+			await this.notificationService.createNotification(me.id, 'createToken', {});
```

Comment:
ここは `await` する必要はなさそうです。
```suggestion
			this.notificationService.createNotification(me.id, 'createToken', {});
```

Reply:
確かに不要ですね、消しておきます。